backend/capellacollab/core/database/__main__.py

```diff
@@ -51,6 +54,9 @@ def migrate_db():
             command.stamp(alembic_cfg, "head")
             initialize_admin_user()
             initialize_default_repository()
+        
+        if not tools_exist:
+            create_tools()
```

Comment:
This should be done by the migration scripts and the alembic scripts should also be the only location where that should happen.